packages/contracts/tests-ts/src/constants/merkle-tree.ts

```diff
@@ -18,7 +18,7 @@ export const FT_DECIMALS = "000000";
 
 export const NEAR_DECIMALS = "000000000000000000000000";
 
-export const HAPI_ONE_TESTNET = "proxy.hapi-npo.testnet";
+export const HAPI_ONE_TESTNET = "dev-1681141193055-11311550135061";
```

Comment:
Why did we use this account? Looks to me like the address is not a constant anymore and depends on the value of `hapi.accountId` set in registry.ts

Exclude this constant